File: Iniciante/URI 1035.hs
main = do
    line <- getLine
    let (a:b:c:d:_) = map (read :: String -> Int) $ words line
    if b > c && d > a && c > 0 && d > 0 && (c + d) > (a + b) && a `mod` 2 == 0
        then putStrLn $ "Valores aceitos"
    else
        putStrLn $ "Valores nao aceitos"

File: Iniciante/URI 1035.lua
line = {}
n = 0
for i in string.gmatch(io.read("*l"), "%S+") do
   n = n + 1
   line[n] = i
end
a = tonumber(line[1])
b = tonumber(line[2])
c = tonumber(line[3])
d = tonumber(line[4])

if (b > c and d > a and c > 0 and d > 0 and (c + d) > (a + b) and a % 2 == 0) then
	print('Valores aceitos')
else
	print('Valores nao aceitos')
end